source/PlayerInfo.cpp

```diff
@@ -551,6 +551,10 @@ void PlayerInfo::IncrementDate()
 			Messages::Add("You failed to meet the deadline for the mission \"" + mission.Name() + "\"."
 				, Messages::Importance::Highest);
 
+	// Do any daily mission actions.
+	for(Mission &mission : missions)
```

Comment:
There's already a `for(Mission &mission : missions)` above this one. No need to iterate over the same list twice; just add the `mission.Do(Mission::DAILY, *this);` line into the loop above this.

Reply:
done